2024-05-10 10:43:18.527 - DBeaver 24.0.3.202404211624 is starting
2024-05-10 10:43:18.527 - OS: Windows 10 10.0 (amd64)
2024-05-10 10:43:18.528 - Java version: 17.0.6 by Eclipse Adoptium (64bit)
2024-05-10 10:43:18.528 - Install path: 'H:\workspace\oracle\DBeaver'
2024-05-10 10:43:18.528 - Instance path: 'file:/H:/workspace/multi/04_db/'
2024-05-10 10:43:18.530 - Memory available 64Mb/1024Mb
2024-05-10 10:43:18.540 - Create display
2024-05-10 10:43:18.737 - Initialize desktop platform...
2024-05-10 10:43:18.924 - BounceCastle bundle found. Use JCE provider BC
> Start Core File Systems [org.eclipse.core.filesystem 1.10.200.v20231102-0934]
> Start Core Resource Management [org.eclipse.core.resources 3.20.0.v20231102-0934]
> Start 팀 지원 핵심 [org.eclipse.team.core 3.10.200.v20231106-1240]
> Start DBeaver UI Navigator [org.jkiss.dbeaver.ui.navigator 1.0.129.202404211624]
2024-05-10 10:43:19.302 - Initialize base platform...
2024-05-10 10:43:19.344 - Platform initialized (606ms)
> Start DBeaver UI Editors - Base [org.jkiss.dbeaver.ui.editors.base 1.0.129.202404211624]
2024-05-10 10:43:19.528 - Starting instance server at http://localhost:6233
2024-05-10 10:43:19.551 - Run workbench
> Start Eclipse e4 작업영역 SWT [org.eclipse.e4.ui.workbench.swt 0.17.200.v20231030-2045]
> Start Eclipse Application Services [org.eclipse.e4.core.services 2.4.200.v20231103-2012]
> Start Eclipse UI Application Services [org.eclipse.e4.ui.services 1.6.200.v20231030-2045]
> Start Eclipse e4 Progress View [org.eclipse.e4.ui.progress 0.4.200.v20231030-2045]
> Start Eclipse Bindings Support [org.eclipse.e4.ui.bindings 0.14.200.v20231030-2045]
> Start Eclipse e4 core commands [org.eclipse.e4.core.commands 1.1.200.v20231108-1058]
> Start EMF XML/XMI Persistence [org.eclipse.emf.ecore.xmi 2.36.0.v20231002-1156]
> Start Eclipse CSS SWT Theme Support [org.eclipse.e4.ui.css.swt.theme 0.14.200.v20230913-1724]
> Start Eclipse e4 작업영역 기능 추가 [org.eclipse.e4.ui.workbench.addons.swt 1.5.200.v20231106-1600]
> Start Eclipse e4 작업영역 SWT 렌더러 [org.eclipse.e4.ui.workbench.renderers.swt 0.16.200.v20231030-2045]
> Start Eclipse Dependency Injection Extensions [org.eclipse.e4.core.di.extensions 0.18.100.v20231103-2012]
> Start Eclipse CSS SWT Support [org.eclipse.e4.ui.css.swt 0.15.200.v20231025-0632]
2024-05-10 10:43:30.398 - Configure workbench window
> Start DBeaver SQL Model [org.jkiss.dbeaver.model.sql 1.0.119.202404211624]
2024-05-10 10:43:30.678 - Total database drivers: 112 (112)
> Start 도움말 시스템 코어 [org.eclipse.help 3.10.200.v20231106-1240]
2024-05-10 10:43:30.854 - Create actions
2024-05-10 10:43:30.874 - Disable Eclipse action set 'org.eclipse.ui.edit.text.actionSet.annotationNavigation'
2024-05-10 10:43:30.876 - Disable Eclipse action set 'org.eclipse.ui.WorkingSetActionSet'
2024-05-10 10:43:30.876 - Disable Eclipse action set 'org.eclipse.ui.actionSet.openFiles'
2024-05-10 10:43:30.876 - Create workbench actions
> Start jna [com.sun.jna 5.13.0.v20230812-1000]
2024-05-10 10:43:32.405 - Initialize workbench window
2024-05-10 10:43:32.416 - Finish initialization
> Start 기본 문서 편집기 [org.eclipse.ui.editors 3.17.100.v20231106-1600]
> Start 텍스트 편집기 프레임워크 [org.eclipse.ui.workbench.texteditor 3.17.200.v20231106-1600]
> Start DBeaver UI Editors - Data [org.jkiss.dbeaver.ui.editors.data 1.0.129.202404211624]
Warning: NLS unused message: actions_navigator__objects in: org.jkiss.dbeaver.ui.editors.sql.internal.SQLEditorMessages
> Start DBeaver UI Editors - SQL [org.jkiss.dbeaver.ui.editors.sql 1.0.129.202404211624]
> Start DBeaver SQL Terminal [org.jkiss.dbeaver.ui.editors.sql.terminal 1.0.105.202404211624]
> Start 핵심 비교 지원 [org.eclipse.compare.core 3.8.300.v20231106-1240]
> Start 인터넷 연결 관리 사용자 인터페이스 [org.eclipse.ui.net 1.5.200.v20231106-1240]
> Start Internet Connection Management [org.eclipse.core.net 1.5.200.v20231106-1240]
> Start DBeaver Sample Database [org.jkiss.dbeaver.ext.sample.database 1.0.166.202404211624]
> Start DBeaver Tip of the day [org.jkiss.dbeaver.ext.ui.tipoftheday 1.0.135.202404211624]
> Start GEF Classic Draw2d [org.eclipse.draw2d 3.14.100.202311222042]
> Start GEF Classic GEF (MVC) [org.eclipse.gef 3.16.0.202311222042]
> Start DBeaver ERD UI [org.jkiss.dbeaver.erd.ui 3.0.89.202404211624]
> Start DBeaver ERD Model [org.jkiss.dbeaver.erd.model 1.0.89.202404211624]
> Start DBeaver Dashboard UI [org.jkiss.dbeaver.ui.dashboard 1.0.126.202404211624]
> Start DBeaver Dashboards Model [org.jkiss.dbeaver.model.dashboard 1.0.2.202404211624]
> Start DBeaver JDBC Model [org.jkiss.dbeaver.model.jdbc 1.0.22.202404211624]
> Start DBeaver Oracle Model [org.jkiss.dbeaver.ext.oracle 2.1.214.202404211624]
2024-05-10 10:43:35.481 - Core Exception caught while reading saved collapsed folding positions
org.eclipse.core.internal.resources.ResourceException(/General/Scripts/0509HW.sql)[368]: java.lang.Exception: Resource '/General/Scripts/0509HW.sql' does not exist.
	at org.eclipse.core.internal.resources.ResourceException.provideStackTrace(ResourceException.java:42)
	at org.eclipse.core.internal.resources.ResourceException.<init>(ResourceException.java:38)
	at org.eclipse.core.internal.resources.Resource.checkExists(Resource.java:367)
	at org.eclipse.core.internal.resources.Resource.checkAccessible(Resource.java:241)
	at org.eclipse.core.internal.resources.Resource.checkAccessibleAndLocal(Resource.java:247)
	at org.eclipse.core.internal.resources.Resource.getPersistentProperty(Resource.java:1172)
	at org.jkiss.dbeaver.ui.editors.sql.syntax.SQLReconcilingStrategy.getSavedCollapsedAnnotationsOffsets(SQLReconcilingStrategy.java:126)
	at org.jkiss.dbeaver.ui.editors.sql.syntax.SQLReconcilingStrategy.reconcile(SQLReconcilingStrategy.java:250)
	at org.jkiss.dbeaver.ui.editors.sql.syntax.SQLReconcilingStrategy.onDataSourceChange(SQLReconcilingStrategy.java:188)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditorSourceViewerConfiguration.onDataSourceChange(SQLEditorSourceViewerConfiguration.java:429)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditorBase.reloadSyntaxRules(SQLEditorBase.java:841)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor.reloadSyntaxRules(SQLEditor.java:3372)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor.lambda$20(SQLEditor.java:2209)
	at org.jkiss.dbeaver.ui.UIExecutionQueue.executeInUI(UIExecutionQueue.java:68)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:132)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4046)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3662)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:648)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:555)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:173)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.start(DBeaverApplication.java:295)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:208)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.invokeFramework(DBeaverLauncher.java:647)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.basicRun(DBeaverLauncher.java:607)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.run(DBeaverLauncher.java:1464)
> Start 콘솔 [org.eclipse.ui.console 3.13.100.v20230914-0751]
> Start 디버그 핵심 [org.eclipse.debug.core 3.21.200.v20231021-1513]
2024-05-10 10:43:36.130 - Core Exception caught while reading saved collapsed folding positions
org.eclipse.core.internal.resources.ResourceException(/General/Scripts/0509HW.sql)[368]: java.lang.Exception: Resource '/General/Scripts/0509HW.sql' does not exist.
	at org.eclipse.core.internal.resources.ResourceException.provideStackTrace(ResourceException.java:42)
	at org.eclipse.core.internal.resources.ResourceException.<init>(ResourceException.java:38)
	at org.eclipse.core.internal.resources.Resource.checkExists(Resource.java:367)
	at org.eclipse.core.internal.resources.Resource.checkAccessible(Resource.java:241)
	at org.eclipse.core.internal.resources.Resource.checkAccessibleAndLocal(Resource.java:247)
	at org.eclipse.core.internal.resources.Resource.getPersistentProperty(Resource.java:1172)
	at org.jkiss.dbeaver.ui.editors.sql.syntax.SQLReconcilingStrategy.getSavedCollapsedAnnotationsOffsets(SQLReconcilingStrategy.java:126)
	at org.jkiss.dbeaver.ui.editors.sql.syntax.SQLReconcilingStrategy.reconcile(SQLReconcilingStrategy.java:250)
	at org.jkiss.dbeaver.ui.editors.sql.syntax.SQLReconcilingStrategy.onDataSourceChange(SQLReconcilingStrategy.java:188)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditorSourceViewerConfiguration.onDataSourceChange(SQLEditorSourceViewerConfiguration.java:429)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditorBase.reloadSyntaxRules(SQLEditorBase.java:841)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor.reloadSyntaxRules(SQLEditor.java:3372)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor.onDataSourceChange(SQLEditor.java:2898)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor.onDataSourceChange(SQLEditor.java:2893)
	at org.jkiss.dbeaver.ui.UIExecutionQueue.executeInUI(UIExecutionQueue.java:68)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:132)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4046)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3662)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:648)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:555)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:173)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.start(DBeaverApplication.java:295)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:208)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.invokeFramework(DBeaverLauncher.java:647)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.basicRun(DBeaverLauncher.java:607)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.run(DBeaverLauncher.java:1464)
> Start DBeaver UI Editors - Connections [org.jkiss.dbeaver.ui.editors.connection 1.0.120.202404211624]
2024-05-10 10:43:37.831 - Open [https://dbeaver.io/product/version.xml]
2024-05-10 10:43:38.378 - Event loop exception
org.eclipse.swt.SWTException: Widget is disposed
	at org.eclipse.swt.SWT.error(SWT.java:4918)
	at org.eclipse.swt.SWT.error(SWT.java:4833)
	at org.eclipse.swt.SWT.error(SWT.java:4804)
	at org.eclipse.swt.widgets.Widget.error(Widget.java:450)
	at org.eclipse.swt.widgets.Widget.checkWidget(Widget.java:369)
	at org.eclipse.swt.custom.CTabFolder.setFont(CTabFolder.java:2701)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer.applyCurrentPresentationThemeSettings(ResultSetViewer.java:451)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer.access$4(ResultSetViewer.java:449)
	at org.eclipse.ui.internal.PendingSyncExec.run(PendingSyncExec.java:68)
	at org.eclipse.ui.internal.UILockListener.doPendingWork(UILockListener.java:172)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:132)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4046)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3662)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:648)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:555)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:173)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.start(DBeaverApplication.java:295)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:208)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.invokeFramework(DBeaverLauncher.java:647)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.basicRun(DBeaverLauncher.java:607)
	at org.jkiss.dbeaver.launcher.DBeaverLauncher.run(DBeaverLauncher.java:1464)
> Start DBeaver Tasks - Native Tools UI [org.jkiss.dbeaver.tasks.native.ui 1.0.107.202404211624]
> Start DBeaver Oracle UI [org.jkiss.dbeaver.ext.oracle.ui 1.0.116.202404211624]
> Start DBeaver Tasks - General UI [org.jkiss.dbeaver.tasks.ui 1.0.107.202404211624]
2024-05-10 10:43:38.857 - Connect with 'localhost' (oracle_thin-18f3c7e490f-66e9f76381d1fe84)
2024-05-10 10:43:38.925 - Execution context opened (localhost; Main; 0)
> Start DBeaver SQL Language Semantic Model [org.jkiss.dbeaver.model.lsm 1.0.24.202404211624]
2024-05-10 10:43:40.319 - Execution context opened (localhost; Metadata; 1)
2024-05-10 10:43:40.446 - Connected (oracle_thin-18f3c7e490f-66e9f76381d1fe84, Oracle JDBC driver [23.2.0.0.0])
2024-05-10 10:43:40.529 - Execution context opened (localhost; SQLEditor <day04_oracle_scott.sql>; 2)
2024-05-10 10:44:29.515 - Execution context opened (localhost; SQLEditor <day01_oracle_scott.sql>; 3)
2024-05-10 10:44:32.443 - Execution context opened (localhost; SQLEditor <Script-7.sql>; 4)
> Start DBeaver MySQL UI Support [org.jkiss.dbeaver.ext.mysql.ui 1.0.121.202404211624]
2024-05-10 11:01:31.431 - Connect with 'localhost 2' (mysql8-18f3c4755fa-48d44911c1ae78cb)
> Start DBeaver MySQL Support [org.jkiss.dbeaver.ext.mysql 2.1.214.202404211624]
2024-05-10 11:01:31.452 - Execution context opened (localhost 2; Main; 5)
> Start DBeaver GIS Support [org.jkiss.dbeaver.data.gis 1.0.153.202404211624]
2024-05-10 11:01:31.762 - Execution context opened (localhost 2; Metadata; 6)
2024-05-10 11:01:31.905 - Connected (mysql8-18f3c4755fa-48d44911c1ae78cb, MySQL Connector/J [mysql-connector-j-8.2.0 (Revision: 06a1f724497fd81c6a659131fda822c9e5085b6c)])
2024-05-10 11:01:31.946 - Execution context opened (localhost 2; SQLEditor <Script-8.sql>; 7)
> Start DBeaver Data Transfer [org.jkiss.dbeaver.data.transfer 2.0.113.202404211624]
2024-05-10 11:02:55.887 - Error while analyzing query text: CREATE TABLE 
java.lang.NullPointerException: Cannot invoke "org.jkiss.dbeaver.model.stm.STMTreeNode.getNodeName()" because "node" is null
	at org.jkiss.dbeaver.ui.editors.sql.semantics.SQLQueryModelRecognizer.collectIdentifier(SQLQueryModelRecognizer.java:875)
	at org.jkiss.dbeaver.ui.editors.sql.semantics.SQLQueryModelRecognizer.collectQualifiedName(SQLQueryModelRecognizer.java:972)
	at org.jkiss.dbeaver.ui.editors.sql.semantics.SQLQueryModelRecognizer.collectTableName(SQLQueryModelRecognizer.java:953)
	at org.jkiss.dbeaver.ui.editors.sql.semantics.SQLQueryModelRecognizer.traverseForIdentifiers(SQLQueryModelRecognizer.java:468)
	at org.jkiss.dbeaver.ui.editors.sql.semantics.SQLQueryModelRecognizer.recognizeQuery(SQLQueryModelRecognizer.java:678)
	at org.jkiss.dbeaver.ui.editors.sql.semantics.SQLBackgroundParsingJob.doWork(SQLBackgroundParsingJob.java:501)
	at org.jkiss.dbeaver.ui.editors.sql.semantics.SQLBackgroundParsingJob$1.run(SQLBackgroundParsingJob.java:81)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2024-05-10 11:04:15.562 - SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ';
	PW varchar(10),
	NAME varchar(10),
	TEL varchar(10),
	CREATE_DATE timesta' at line 2

Error position: line: 1
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ';
	PW varchar(10),
	NAME varchar(10),
	TEL varchar(10),
	CREATE_DATE timesta' at line 2
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:614)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$2(SQLQueryJob.java:505)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:194)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:524)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:976)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:4155)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:194)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5152)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ';
	PW varchar(10),
	NAME varchar(10),
	TEL varchar(10),
	CREATE_DATE timesta' at line 2
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:767)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:652)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2024-05-10 11:05:26.112 - SQL Error [1146] [42S02]: Table 'scott.member' doesn't exist
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1146] [42S02]: Table 'scott.member' doesn't exist
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:614)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$2(SQLQueryJob.java:505)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:194)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:524)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:976)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:4155)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:194)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5152)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: Table 'scott.member' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:767)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:652)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2024-05-10 11:05:31.497 - SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ';
	PW varchar(10),
	NAME varchar(10),
	TEL varchar(10),
	CREATE_DATE timesta' at line 2
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ';
	PW varchar(10),
	NAME varchar(10),
	TEL varchar(10),
	CREATE_DATE timesta' at line 2
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:614)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$2(SQLQueryJob.java:505)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:194)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:524)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:976)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:4155)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:194)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5152)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ';
	PW varchar(10),
	NAME varchar(10),
	TEL varchar(10),
	CREATE_DATE timesta' at line 2
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:767)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:652)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2024-05-10 11:05:43.606 - SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ';
	PW varchar(10),
	NAME varchar(10),
	TEL varchar(10),
	CREATE_DATE timesta' at line 2
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ';
	PW varchar(10),
	NAME varchar(10),
	TEL varchar(10),
	CREATE_DATE timesta' at line 2
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:614)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$2(SQLQueryJob.java:505)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:194)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:524)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:976)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:4155)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:194)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5152)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ';
	PW varchar(10),
	NAME varchar(10),
	TEL varchar(10),
	CREATE_DATE timesta' at line 2
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:767)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:652)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2024-05-10 11:09:39.454 - Can't get column 'password': Column 'password' not found.
> Start Equinox Provisioning Metadata [org.eclipse.equinox.p2.metadata 2.8.0.v20231102-1258]
2024-05-10 11:11:41.641 - Connect with 'localhost 3' (oracle_thin-18f3cd72e0c-5d72d8081e0b6b19)
2024-05-10 11:11:41.641 - Execution context opened (localhost 3; Main; 8)
2024-05-10 11:11:41.681 - Execution context opened (localhost 3; Metadata; 9)
2024-05-10 11:11:41.728 - Connected (oracle_thin-18f3cd72e0c-5d72d8081e0b6b19, Oracle JDBC driver [23.2.0.0.0])
2024-05-10 11:11:41.764 - Execution context opened (localhost 3; SQLEditor <Script-8.sql>; 10)
2024-05-10 11:11:43.245 - Execution context closed (localhost 3, 10)
2024-05-10 11:11:46.776 - Execution context opened (localhost 3; SQLEditor <Script-8.sql>; 11)
2024-05-10 11:11:48.255 - Execution context closed (localhost 3, 11)
2024-05-10 11:11:52.361 - Connect with 'localhost 4' (oracle_thin-18f3cfe8351-3c17426f1dacd56f)
2024-05-10 11:11:52.362 - Execution context opened (localhost 4; Main; 12)
2024-05-10 11:11:52.404 - Execution context opened (localhost 4; Metadata; 13)
2024-05-10 11:11:52.444 - Connected (oracle_thin-18f3cfe8351-3c17426f1dacd56f, Oracle JDBC driver [23.2.0.0.0])
2024-05-10 11:11:52.483 - Execution context opened (localhost 4; SQLEditor <Script-9.sql>; 14)
2024-05-10 11:11:55.043 - Connect with 'localhost 5' (oracle_thin-18f3d20ecd6-76790cb35d739ef5)
2024-05-10 11:11:55.045 - Execution context opened (localhost 5; Main; 15)
2024-05-10 11:11:55.131 - ORA-01017: invalid username/password; logon denied

org.jkiss.dbeaver.model.exec.DBCConnectException: ORA-01017: invalid username/password; logon denied

	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:265)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.openConnection(OracleDataSource.java:176)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:106)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:119)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.<init>(OracleDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.OracleDataSourceProvider.openDataSource(OracleDataSourceProvider.java:143)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1366)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect0(DataSourceDescriptor.java:1243)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1033)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:78)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:702)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:603)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:598)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:1710)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:819)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:1041)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:443)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:462)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTHWithO5Logon(T4CTTIoauthenticate.java:1628)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:1375)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:1322)
	at oracle.jdbc.driver.T4CConnection.authenticateWithPassword(T4CConnection.java:1516)
	at oracle.jdbc.driver.T4CConnection.authenticateUserForLogon(T4CConnection.java:1459)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:828)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1132)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:837)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:664)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:109)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:83)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:221)
	... 15 more
2024-05-10 11:11:59.442 - Connect with 'localhost 6' (oracle_thin-18f3d7ffe3e-764742beee258caf)
2024-05-10 11:11:59.443 - Execution context opened (localhost 6; Main; 16)
2024-05-10 11:11:59.509 - ORA-01017: invalid username/password; logon denied

org.jkiss.dbeaver.model.exec.DBCConnectException: ORA-01017: invalid username/password; logon denied

	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:265)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.openConnection(OracleDataSource.java:176)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:106)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:119)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.<init>(OracleDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.OracleDataSourceProvider.openDataSource(OracleDataSourceProvider.java:143)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1366)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect0(DataSourceDescriptor.java:1243)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1033)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:78)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:702)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:603)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:598)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:1710)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:819)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:1041)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:443)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:462)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTHWithO5Logon(T4CTTIoauthenticate.java:1628)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:1375)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:1322)
	at oracle.jdbc.driver.T4CConnection.authenticateWithPassword(T4CConnection.java:1516)
	at oracle.jdbc.driver.T4CConnection.authenticateUserForLogon(T4CConnection.java:1459)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:828)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1132)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:837)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:664)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:109)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:83)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:221)
	... 15 more
2024-05-10 11:12:03.311 - Connect with 'localhost 7' (oracle_thin-18f3d81da13-724e6d90f262c545)
2024-05-10 11:12:03.311 - Execution context opened (localhost 7; Main; 17)
2024-05-10 11:12:03.387 - ORA-01017: invalid username/password; logon denied

org.jkiss.dbeaver.model.exec.DBCConnectException: ORA-01017: invalid username/password; logon denied

	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:265)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.openConnection(OracleDataSource.java:176)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:106)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:119)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.<init>(OracleDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.OracleDataSourceProvider.openDataSource(OracleDataSourceProvider.java:143)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1366)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect0(DataSourceDescriptor.java:1243)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1033)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:78)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:702)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:603)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:598)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:1710)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:819)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:1041)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:443)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:462)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTHWithO5Logon(T4CTTIoauthenticate.java:1628)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:1375)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:1322)
	at oracle.jdbc.driver.T4CConnection.authenticateWithPassword(T4CConnection.java:1516)
	at oracle.jdbc.driver.T4CConnection.authenticateUserForLogon(T4CConnection.java:1459)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:828)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1132)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:837)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:664)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:109)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:83)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:221)
	... 15 more
2024-05-10 11:12:06.611 - Execution context closed (localhost 4, 14)
2024-05-10 15:12:31.088 - Execution context opened (localhost; SQLEditor <Script-13.sql>; 18)
2024-05-10 15:13:38.871 - Can't get column 'EXTERNAL_NAME': ORA-17006: 열 이름이 부적합합니다.
2024-05-10 15:13:38.871 - Can't get column 'ACCOUNT_STATUS': ORA-17006: 열 이름이 부적합합니다.
2024-05-10 15:13:38.872 - Can't get column 'LOCK_DATE': ORA-17006: 열 이름이 부적합합니다.
2024-05-10 15:13:38.872 - Can't get column 'EXPIRY_DATE': ORA-17006: 열 이름이 부적합합니다.
2024-05-10 15:13:38.872 - Can't get column 'DEFAULT_TABLESPACE': ORA-17006: 열 이름이 부적합합니다.
2024-05-10 15:13:38.872 - Can't get column 'TEMPORARY_TABLESPACE': ORA-17006: 열 이름이 부적합합니다.
2024-05-10 15:13:38.872 - Can't get column 'PROFILE': ORA-17006: 열 이름이 부적합합니다.
2024-05-10 15:13:38.872 - Can't get column 'INITIAL_RSRC_CONSUMER_GROUP': ORA-17006: 열 이름이 부적합합니다.
2024-05-10 15:14:17.764 - Execution context closed (localhost, 0)
2024-05-10 15:14:17.775 - Execution context closed (localhost, 1)
2024-05-10 15:14:17.786 - Execution context closed (localhost, 2)
2024-05-10 15:14:17.797 - Execution context closed (localhost, 3)
2024-05-10 15:14:17.808 - Execution context closed (localhost, 4)
2024-05-10 15:14:17.819 - Execution context closed (localhost, 18)
2024-05-10 15:14:17.819 - Disconnected (oracle_thin-18f3c7e490f-66e9f76381d1fe84)
2024-05-10 15:14:19.781 - Connect with 'localhost' (oracle_thin-18f3c7e490f-66e9f76381d1fe84)
2024-05-10 15:14:19.782 - Execution context opened (localhost; Main; 19)
2024-05-10 15:14:19.809 - Execution context opened (localhost; Metadata; 20)
2024-05-10 15:14:19.842 - Connected (oracle_thin-18f3c7e490f-66e9f76381d1fe84, Oracle JDBC driver [23.2.0.0.0])
2024-05-10 15:14:19.875 - Execution context opened (localhost; SQLEditor <day04_oracle_scott.sql>; 21)
2024-05-10 15:14:19.875 - Execution context opened (localhost; SQLEditor <day01_oracle_scott.sql>; 22)
2024-05-10 15:14:19.876 - Execution context opened (localhost; SQLEditor <Script-7.sql>; 23)
2024-05-10 15:14:19.876 - Execution context opened (localhost; SQLEditor <Script-13.sql>; 24)
2024-05-10 15:16:06.241 - Execution context opened (localhost; SQLEditor <Script-14.sql>; 25)
2024-05-10 15:16:25.468 - org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [942] [42000]: ORA-00942: table or view does not exist

	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:614)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$2(SQLQueryJob.java:505)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:194)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:524)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.run(SQLQueryJob.java:239)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: ORA-00942: table or view does not exist

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:702)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:608)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1335)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:1041)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:443)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:533)
	at oracle.jdbc.driver.T4CStatement.doOall8(T4CStatement.java:176)
	at oracle.jdbc.driver.T4CStatement.executeForRows(T4CStatement.java:1305)
	at oracle.jdbc.driver.OracleStatement.executeSQLStatement(OracleStatement.java:1877)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1520)
	at oracle.jdbc.driver.OracleStatement.executeInternal(OracleStatement.java:2557)
	at oracle.jdbc.driver.OracleStatement.execute(OracleStatement.java:2506)
	at oracle.jdbc.driver.OracleStatementWrapper.execute(OracleStatementWrapper.java:334)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 7 more
Caused by: Error : 942, Position : 11, SQL = DROP TABLE MEMBER CASCADE CONSTRAINT, Original SQL = DROP TABLE MEMBER CASCADE CONSTRAINT, Error Message = ORA-00942: table or view does not exist

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:710)
	... 21 more
2024-05-10 15:17:42.334 - 	Invalidate context 'Main' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (BEFORE_INVALIDATE)
2024-05-10 15:17:42.346 - Execution context closed (localhost, 19)
2024-05-10 15:17:42.346 - 	Invalidate context 'Metadata' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (BEFORE_INVALIDATE)
2024-05-10 15:17:42.357 - Execution context closed (localhost, 20)
2024-05-10 15:17:42.357 - 	Invalidate context 'SQLEditor <day04_oracle_scott.sql>' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (BEFORE_INVALIDATE)
2024-05-10 15:17:42.368 - Execution context closed (localhost, 21)
2024-05-10 15:17:42.368 - 	Invalidate context 'SQLEditor <day01_oracle_scott.sql>' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (BEFORE_INVALIDATE)
2024-05-10 15:17:42.379 - Execution context closed (localhost, 22)
2024-05-10 15:17:42.379 - 	Invalidate context 'SQLEditor <Script-7.sql>' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (BEFORE_INVALIDATE)
2024-05-10 15:17:42.390 - Execution context closed (localhost, 23)
2024-05-10 15:17:42.390 - 	Invalidate context 'SQLEditor <Script-13.sql>' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (BEFORE_INVALIDATE)
2024-05-10 15:17:42.402 - Execution context closed (localhost, 24)
2024-05-10 15:17:42.402 - 	Invalidate context 'SQLEditor <Script-14.sql>' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (BEFORE_INVALIDATE)
2024-05-10 15:17:42.414 - Execution context closed (localhost, 25)
2024-05-10 15:17:42.414 - 	Invalidate context 'Main' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (INVALIDATE)
2024-05-10 15:17:42.443 - 	Invalidate context 'Metadata' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (INVALIDATE)
2024-05-10 15:17:42.472 - 	Invalidate context 'SQLEditor <day04_oracle_scott.sql>' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (INVALIDATE)
2024-05-10 15:17:42.497 - 	Invalidate context 'SQLEditor <day01_oracle_scott.sql>' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (INVALIDATE)
2024-05-10 15:17:42.523 - 	Invalidate context 'SQLEditor <Script-7.sql>' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (INVALIDATE)
2024-05-10 15:17:42.547 - 	Invalidate context 'SQLEditor <Script-13.sql>' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (INVALIDATE)
2024-05-10 15:17:42.574 - 	Invalidate context 'SQLEditor <Script-14.sql>' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (INVALIDATE)
2024-05-10 15:17:42.602 - 	Invalidate context 'Main' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (AFTER_INVALIDATE)
2024-05-10 15:17:42.602 - 	Invalidate context 'Metadata' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (AFTER_INVALIDATE)
2024-05-10 15:17:42.602 - 	Invalidate context 'SQLEditor <day04_oracle_scott.sql>' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (AFTER_INVALIDATE)
2024-05-10 15:17:42.602 - 	Invalidate context 'SQLEditor <day01_oracle_scott.sql>' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (AFTER_INVALIDATE)
2024-05-10 15:17:42.602 - 	Invalidate context 'SQLEditor <Script-7.sql>' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (AFTER_INVALIDATE)
2024-05-10 15:17:42.603 - 	Invalidate context 'SQLEditor <Script-13.sql>' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (AFTER_INVALIDATE)
2024-05-10 15:17:42.603 - 	Invalidate context 'SQLEditor <Script-14.sql>' for oracle_thin-18f3c7e490f-66e9f76381d1fe84 (AFTER_INVALIDATE)
2024-05-10 15:17:42.623 - Connections reopened: 7 (of 7)
> Start Data Transfer UI [org.jkiss.dbeaver.data.transfer.ui 1.0.113.202404211624]
2024-05-10 15:17:46.284 - Service 'org.jkiss.dbeaver.runtime.ui.UIServiceSecurity' not found
2024-05-10 15:17:46.330 - Unrecognized Oracle client version at H:/workspace/oracle/DBeaver/jre
2024-05-10 15:18:13.791 - Connect with 'localhost 5' (oracle_thin-18f3d20ecd6-76790cb35d739ef5)
2024-05-10 15:18:13.792 - Execution context opened (localhost 5; Main; 26)
2024-05-10 15:18:13.821 - ORA-01017: invalid username/password; logon denied

org.jkiss.dbeaver.model.exec.DBCConnectException: ORA-01017: invalid username/password; logon denied

	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:265)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.openConnection(OracleDataSource.java:176)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:106)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:119)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.model.OracleDataSource.<init>(OracleDataSource.java:94)
	at org.jkiss.dbeaver.ext.oracle.OracleDataSourceProvider.openDataSource(OracleDataSourceProvider.java:143)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1366)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect0(DataSourceDescriptor.java:1243)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1033)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:78)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:115)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:702)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:603)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:598)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:1710)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:819)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:1041)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:443)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:462)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTHWithO5Logon(T4CTTIoauthenticate.java:1628)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:1375)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:1322)
	at oracle.jdbc.driver.T4CConnection.authenticateWithPassword(T4CConnection.java:1516)
	at oracle.jdbc.driver.T4CConnection.authenticateUserForLogon(T4CConnection.java:1459)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:828)
	at oracle.jdbc.driver.PhysicalConnection.connect(PhysicalConnection.java:1132)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:104)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:837)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:664)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:109)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCConnectionOpener.run(JDBCConnectionOpener.java:83)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:221)
	... 15 more
2024-05-10 15:18:19.413 - Service 'org.jkiss.dbeaver.runtime.ui.UIServiceSecurity' not found
2024-05-10 15:18:32.525 - Connect with 'localhost 5' (oracle_thin-18f3d20ecd6-76790cb35d739ef5)
2024-05-10 15:18:32.525 - Execution context opened (localhost 5; Main; 27)
2024-05-10 15:18:32.552 - Execution context opened (localhost 5; Metadata; 28)
2024-05-10 15:18:32.587 - Connected (oracle_thin-18f3d20ecd6-76790cb35d739ef5, Oracle JDBC driver [23.2.0.0.0])
2024-05-10 15:18:35.863 - Execution context closed (localhost 5, 27)
2024-05-10 15:18:35.874 - Execution context closed (localhost 5, 28)
2024-05-10 15:18:35.874 - Disconnected (oracle_thin-18f3d20ecd6-76790cb35d739ef5)
2024-05-10 18:16:49.423 - Execution context closed (localhost, 24)
2024-05-10 18:16:49.466 - Execution context closed (localhost, 25)
2024-05-10 18:16:49.511 - Execution context closed (localhost 2, 7)
2024-05-10 18:16:49.552 - Execution context closed (localhost, 21)
2024-05-10 18:16:49.591 - Execution context closed (localhost, 23)
2024-05-10 18:16:49.627 - Execution context closed (localhost, 22)
2024-05-10 18:16:50.018 - DBeaver is stopping
2024-05-10 18:16:50.020 - Stop instance server
2024-05-10 18:16:51.029 - Instance server has been stopped
2024-05-10 18:16:51.029 - DBeaver shutdown completed
